supp2u/src/components/customerviews/CustomerView.js

```diff
@@ -1,31 +1,58 @@
-import React, { useState } from "react";
-import Axios from "axios";
-
-const CustomerView = () => {
-  const [customerListView, setCustomerListView] = useState([{}]);
-
-  let fetchData = () => {
-    Axios.get(
-      // `${process.env.REACT_APP_BACKEND_URL}users/1/customers`
-      `http://localhost:3001/api/v1/users/1/customers`
-    )
-      .then(res => {
-        console.log('res', res);
-        
-      })
-      .catch(error => {
-        alert(error);
-      });
-  }
-  fetchData();
+import React, { useState, useEffect } from 'react'
+import Axios from 'axios'
+import Reviews from '../Reviews';
 
 
+function CustomerView() {
+
+  const [customer, setCustomer] = useState([{}])
+  const [review, setReview] = useState([{}])
+
+// This pulls the user_id from local storage
+  let user_id = localStorage.getItem("user_id");
+
+  useEffect(() => {
+    //api call to get the information on the users customer status
+    Axios.get(`${process.env.REACT_APP_BACKEND_URL}users/${user_id}/customers`)
+        .then(res =>{
+            setCustomer(res.data)
+            // api call to get the reviews connect to the customer it must be nested in order to get the id of the customer
```

Comment:
Helpful comments throughout the component. I know I'll appreciate this not having spent too much time with these up to this point so thanks 🔥